js/kubewarden/host_capabilities/crypto/types.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Certificate encoding types
+ */
+export type CertificateEncoding = 'Der' | 'Pem';
+
+/**
+ * Certificate represents a x509 certificate
+ */
+export interface Certificate {
+  encoding: CertificateEncoding;
+  data: number[];
+}
+
+/**
+ * CertificateVerificationRequest holds information about a certificate and
+ * a chain to validate it with
+ */
+export interface CertificateVerificationRequest {
+  cert: Certificate;
+  cert_chain: Certificate[];
```

Comment:
Both `cert_chain` and `not_after` are optional, hence they may be missing from the JSON that a policy sends to the host.

It seems that the `cert_chain` may be a None or an empty list, but for the `not_after` we need to not provide it to skip it.